spec/Organization.tex

```diff
@@ -137,17 +132,15 @@
 
 \item
 Chapter~\ref{Standard_Modules}, Standard Modules, describes the
-standard modules that are provided with the Chapel language.
+structure of standard modules that are provided with the Chapel language
+and the \chpl{Base} standard module.
```

Comment:
Ah, I see.  It's a lot smaller and still has the Base module in it (which isn't technically a standard module)